TorchSharp.PyBridge/PyTorchUnpickler.cs

```diff
@@ -182,15 +191,43 @@ class ParameterObjectConstructor : IObjectConstructor {
             }
         }
 
+        internal record TensorConstructorArgs
+        {
+            public int archiveIndex { get; init; }
+
+            public Stream data { get; init; }
+
+            public torch.ScalarType dtype { get; init; }
+
+            public int storageOffset { get; init; }
+
+            public long[] shape { get; init; }
+
+            public long[] stride { get; init; }
+
+            public bool requiresGrad { get; init; }
+
+            public torch.Tensor readTensorFromStream() {
+                var temp = torch
+                    .empty(shape, dtype, device: torch.CPU)
+                    .as_strided(shape, stride, storageOffset);
+                temp.ReadBytesFromStream(data);
+                data.Close();
+
+                return temp;
+            }
+        }
```

Comment:
Style: Function names should be PascalCase
Style: Class/Record properties should also be PascalCase (I think I missed a few in my original implementation)
Style: Curly braces shouldn't be on their own line

Reply:
Apologies for disregarding standard C# conventions.

I applied all the suggested styling changes.  Let me know if I missed a spot!

It would be nice to commit a `.editorconfig` file, or maybe there's one, and my IDE (Jetbrains Rider) didn't pick it up.

Reply:
No worries! There is no `.editorconfig` file, but I should probably add it for the future.